editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx

```diff
@@ -131,14 +140,17 @@ function Component(props: Props) {
         </ErrorWrapper>
         <Box>
           <Typography fontWeight={FONT_WEIGHT_BOLD}>Required files</Typography>
-          {fileList.required.map((fileType) => (
-            <ListItem key={fileType.key} disablePadding>
-              <InteractiveFileListItem
-                name={fileType.key}
-                moreInformation={fileType.moreInformation}
-              />
-            </ListItem>
-          ))}
+
+          <List disablePadding>
+            {fileList.required.map((fileType) => (
+              <ListItem key={fileType.key} disablePadding>
+                <InteractiveFileListItem
+                  name={fileType.key}
+                  moreInformation={fileType.moreInformation}
+                />
+              </ListItem>
+            ))}
+          </List>
```

Comment:
:+1: for this a11y fix - I also added an outer `List` and replaced `Typography` with a `ListSubheader` over here https://github.com/theopensystemslab/planx-new/pull/1759/files#diff-36665c16b9d4531b049313c7577008ca7ec35d1a34915ae16f1fc9e55803d3f4R135 

just as heads up so we can keep an eye on this one while rebasing!